juniper_codegen/Cargo.toml

```diff
@@ -22,7 +22,7 @@ regex = "1"
 lazy_static = "1.0.0"
 
 [dev-dependencies]
-juniper = { version = "0.11", path = "../juniper" }
```

Comment:
We publish this so it needs a version, right?

https://crates.io/crates/juniper_codegen

Reply:
Yeah I just temporarily removed it because we need to patch the release Script to bump the version there too. I'll fix that tonight.

Reply:
Changes I landed last night definitely bumped codegen versions. If Yuri base you will see. Also, you'll pick up the CI changes and can see what it would do in the logs

Reply:
It already was rebased to your latest change.

I think the issue was that the dependency was on `0.11` rather than `0.11.1`,  dry run works fine for me now locally. (pushed).